### PR TITLE
Checkout: BrowserInfo: remove AcceptHeader and UserAgent validation to match real scenario

### PR DESCRIPTION
**Pull request by rooque**
## Summary

Change the validation to match the information sent by the drop in component:

```json
"browserInfo": {
    "acceptHeader": "*/*",
    "colorDepth": 30,
    "javaEnabled": false,
    "language": "pt-BR",
    "screenHeight": 900,
    "screenWidth": 1440,
    "timeZoneOffset": 180,
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36"
  }
```

**Fixed issue**:  #305

### CODE REVIEW

File: Adyen/Model/Checkout/BrowserInfo.cs [hunk 1]
```diff
@@ -283,21 +283,21 @@ public override int GetHashCode()
         IEnumerable<System.ComponentModel.DataAnnotations.ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
         {
```
Comment (AlexandrosMor) [inline Adyen/Model/Checkout/BrowserInfo.cs:284]: I think it is better to remove the validation body or the validation function.

**Comment by rooque** [comment]
@AlexandrosMor this way?

File: Adyen/Model/Checkout/BrowserInfo.cs (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen/Model/Checkout/BrowserInfo.cs:284]: yes looks good :)

**Comment by rooque** [comment]
Can we have this merged and then a new release? 

@AlexandrosMor @msilvagarcia @Aleffio @rikterbeek @martinsrenato

File: Adyen/Model/Checkout/BrowserInfo.cs (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen/Model/Checkout/BrowserInfo.cs:284]: Browser info comes from the checkout component so there is no point for the model to make an extra validation

**Pull request merged**